### DISCUSSION TITLE
Looking up title doesnt return any results

### DISCUSSION THREAD

**Discussion by Sockeeeee** [pystardust/ani-cli#1238] [Q&A] [answered]
Hey,
so I've installed ani-cli so far to the point where i can run it in the terminal and type in commands. But if i type in the title of an anime it gets stuck like seen in the picture. If i enter a random name that isn't an anime, it returns "no results found" and quits ani-cli.
![image](https://github.com/pystardust/ani-cli/assets/151607997/585d8fbd-0f30-426e-a806-b5338a8a541c)
I'm pretty sure I installed all dependencies. Using ani-cli -U didnt help. Reinstalling didnt help either. After reinstalling it also didnt say anyhting about missing dependencies. 
Not sure if this is a common issue but if anyone knows a fix i'd appreciate it!

**Comment by 71zenith** [comment, marked as answer]
it is specified in the readme to use windows terminal instead of git bash

**Reply by Sockeeeee** [reply]
Thank you, I saw some conflicting info online that must've been outdated then.
